convert2rhel/actions/system_checks/custom_repos_are_valid.py

```diff
@@ -37,8 +37,8 @@ def run(self):
         super(CustomReposAreValid, self).run()
         logger.task("Prepare: Check if --enablerepo repositories are accessible")
 
-        if not tool_opts.no_rhsm:
-            logger.info("Did not perform the check of repositories due to the use of RHSM for the conversion.")
+        if not tool_opts.enablerepo:
```

Comment:
This way the custom repo accessibility check would fail when using RHSM **and** without the system being pre-registered. At that point the repo the user wants convert2rhel to enable during the conversion is not yet available. One possible solution is to make this check run much later, after sub-man enables the repos needed.